src/com/mitrais/atm/screen/TransactionScreen.java

```diff
@@ -38,22 +43,62 @@ private void selectTransaction(String option, Account account) {
                 transferScreen.show(account);
                 break;
             case "3":
-                welcomeScreen.show();
+                showStatement(account);
                 break;
             case "4":
-                try {
-                    System.out.println("> Account Number: "+ account.getAccountNumber() +
-                            "\n> Balance: $" + account.getBalance());
-                    Thread.sleep(2000);
-                    show(account);
-                } catch (InterruptedException e) {
-                    e.printStackTrace();
-                }
-
+                welcomeScreen.show();
                 break;
             default:
                 show(account);
                 break;
         }
     }
+
+    private void showStatement(Account account) {
+        List<Transaction> transactionList = account.getTransactionList();
+        if(transactionList != null && !transactionList.isEmpty()) {
+            List<Transaction> transactionListFiltered = transactionList.stream()
+                    .sorted(Comparator.comparing(Transaction::getTransactionDate).reversed())
```

Comment:
Great idea, but remember to use Java 8 DateTime API

Reply:
I think the solution is similar with 
1fe1f54